test_tortoise_interpreter.py

```diff
@@ -0,0 +1,52 @@
+from tortoise_interpreter import (
+    sanitize_program,
+    navigate_command_functions,
+    pen_up, pen_colour,
+    )
+
+
+def test_sanatize_program_tiny_tortoise():
+    sanitized_program = sanitize_program([['P', ' ', '3'], ['U'], ['D']])
+    assert sanitized_program == [['P', '3'], ['U'], ['D']]
+
+
+def test_sanatize_program_medium_tortoise():
+    sanitized_program = sanitize_program([['P', ' ', '3'], ['U'], ['D'], ['P', ' ', '3'], ['U'], ['D']])
+    assert sanitized_program == [['P', '3'], ['U'], ['D'], ['P', '3'], ['U'], ['D']]
+
+
+def test_sanatize_program_example_tortoise():
+    sanitized_program = sanitize_program(
+        [
+            ['P', ' ', '2', ' ', ' ', '#', ' ', 's', 'e', 'l', 'e', 'c', 't', ' ', 'p', 'e', 'n', ' ', '2'],
+            ['D', ' ', ' ', ' ', ' ', '#', ' ', 'p', 'e', 'n', ' ', 'd', 'o', 'w', 'n'],
+            ['W', ' ', '2', ' ', ' ', '#', ' ', 'd', 'r', 'a', 'w', ' ', 'w', 'e', 's', 't', ' ', '2', 'c', 'm'],
+            ['N', ' ', '1', ' ', ' ', '#', ' ', 't', 'h', 'e', 'n', ' ', 'n', 'o', 'r', 't', 'h', ' ', '1'],
+            ['E', ' ', '2', ' ', ' ', '#', ' ', 't', 'h', 'e', 'n', ' ', 'e', 'a', 's', 't', ' ', '2'],
+            ['S', ' ', '1', ' ', ' ', '#', ' ', 't', 'h', 'e', 'n', ' ', 'b', 'a', 'c', 'k', ' ', 's', 'o', 'u', 't', 'h'],
+            ['U', ' ', ' ', ' ', ' ', '#', ' ', 'p', 'e', 'n', ' ', 'u', 'p']
+        ]
```

Comment:
This would be more readable if the test code was stored as strings, and then split before passing to `sanitize_program`.